pilot/pkg/networking/plugin/mixer/mixer.go

```diff
@@ -276,7 +271,8 @@ func buildTransport(mesh *meshconfig.MeshConfig, node *model.Proxy) *mccpb.Trans
 
 	networkFailPolicy := &mccpb.NetworkFailPolicy{Policy: policy}
 
-	if annotation, ok := node.Metadata[policyCheckRetriesAnnotation]; ok {
+	networkFailPolicy.MaxRetry = defaultRetries
```

Comment:
Can we _not_ set the config if it matches the hard-coded default in the proxy? This is proliferating config which hurts us in performance.

Reply:
+1 on avoiding config proliferation - unfortunately won't work. If we don't set it - proto3 defaults to 0, so envoy won't know if we want 0 or the built-in default.